lib/src/getContents/index.js

```diff
@@ -14,7 +15,9 @@ function getContents(opt) {
 
     // read and pass full contents
     if (opt.buffer !== false) {
-      return bufferFile(file, cb);
+      // set encoding type if user specifies it
+      var options = 'encoding' in opt ? {'encoding': opt.encoding} : null;
```

Comment:
Why not just `var options = { encoding: opt.encoding }`? If `opt.encoding` is undefined, it'll be passed through as undefined, which is fine.